src/api-explorer/v4-0/ConcurRequest.swagger2.json

```
@@ -0,0 +1,2562 @@
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom2": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom20": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom3": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom4": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom5": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom6": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom7": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom8": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom9": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "expenseId": {
+          "type": "string",
+          "description": "The unique identifier of the expense associated with the allocation"
+        },
+        "percentEdited": {
+          "type": "boolean",
+          "example": false,
+          "description": "Whether the allocation percent has been edited"
+        },
+        "percentage": {
+          "type": "number",
+          "description": "The percentage of the total expense that this allocation represents"
+        },
+        "postedAmount": {
+          "description": "The amount of the allocation calculated with the percentage value multiplied by the posted amount on the expense. This amount is given in the user's currency",
+          "$ref": "#/definitions/Amount"
+        },
+        "systemAllocation": {
+          "type": "boolean",
+          "example": false,
+          "description": "Whether the allocation is a system allocation, usually hidden from the user. If displayed to the user, should be read-only"
```

@quancheng-zhao 
```suggestion
          "description": "Whether the allocation is a system allocation. It is usually hidden from the user. If it is displayed to the user it should be set to read-only"
```
When is it usually hidden from the user? How is it hidden from the user?
How do you set this to read-only?
It's a boolean value -> True = enabled (Allocation is a system allocation), False = disabled (Allocation is not a system allocation).

---

Done. Clarified.